lib/generators/templates/rpush_3_2_5_updates.rb

```diff
@@ -0,0 +1,9 @@
+class Rpush325Updates < ActiveRecord::VERSION::MAJOR >= 5 ? ActiveRecord::Migration[5.0] : ActiveRecord::Migration
```

Comment:
Could you maybe change it to rpush_3_3_0_updates? Since it's kind of a feature, we might go for a 3.3.0 release instead of a patch release.